spec/requests/configuration_templates_spec.rb

```diff
@@ -0,0 +1,30 @@
+RSpec.describe "configuration templates API" do
+  describe "display a configuration template's details" do
+    context "with the user authorized" do
+      it "responds with configuration template details" do
+        config_template = FactoryGirl.create(:configuration_template,
+                                             :ems_id  => 1,
```

Comment:
Even though this may work, setting ids yourself can cause surprises down the road. It may be best to create a real ems and let Rails handle which ids need to be set:

```ruby
FactoryGirl.create(:configuration_template, :ems => FactoryGirl.create(:ems_lenovo), :etc => "etc")
```